test/e2e-test/appconfig_dependency_test.go

```diff
@@ -204,7 +204,7 @@ var _ = Describe("Resource Dependency in an ApplicationConfiguration", func() {
 			func() error {
 				return k8sClient.Get(ctx, inFooKey, inFoo)
 			},
-			time.Second*15, time.Millisecond*500).Should(BeNil())
+			time.Second*60, time.Second*5).Should(BeNil())
```

Comment:
polling interval set to 1s would be better?

Reply:
ok, and then the timeout value should be bigger? Because last time, it failed when timeout is 60s, polling interval is 5s.

Reply:
5s should be fine.